package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
```

Comment:
- [ ] [OPTIONAL] Please add your MIT license to your project and implement is in Readme file.